### DISCUSSION TITLE
No policy to update pivot fields of ManyToMany relationship

### DISCUSSION THREAD

**Discussion by kichetof** [laravel/nova-issues#5049] [Ideas & Feature Requests]
- Laravel Version: 9.39
- Nova Version: 4.18.2
- PHP Version: 8.1.11
- Database Driver & Version: Mysql
- Operating System and Version: macOS Laravel Sail

### Description:
You can authorize user to attach or detach relationship, but you can't authorize a user to update pivot fields without attaching policy.
In my case, user shouldn't be able to attach a new relationship, only update pivot fields on an existing relation.

Could it be possible to have an `updateAttaching{Model}` authorization?

### Detailed steps to reproduce the issue on a fresh Nova installation:

Create a many to many relationship with some pivot fields.
To update pivot fields, you need `attach{Model}` authorization

**Comment by puzzledmonkey** [comment]
seconded - this would be very useful.

thank you!

**Reply by dominik-eller** [reply]
I did not found a solution as well. Would be great to get that.
